src/Aspire.Hosting.Dapr/Utf8JsonWriterExtensions.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+
+namespace Aspire.Hosting.Dapr;
+
+internal static class Uft8JsonWriterExtensions
```

Comment:
Can we move this to Utils or Publishing :)

Reply:
Moved to `Utils`, though had to be made `public` which means being part of the "official" Aspire API.  I don't like the idea of `InternalsVisibleTo` for the Dapr package.  I suppose other 3rd-party hooks might find those methods useful as well.

Reply:
We can leave this for now, I think shared source would be better.